product-scenarios/scenarios-common/src/main/java/org/wso2/am/scenario/test/common/ScenarioTestBase.java

```diff
@@ -162,7 +162,7 @@ public static TenantManagementServiceClient getTenantManagementServiceClient() t
         try {
             authenticatorClient = new AuthenticatorClient(keyManagerURL);
             URL url = new URL(keyManagerURL);
-            String sessionCookie = authenticatorClient.login("admin", "admin", url.getHost());
+            String sessionCookie = authenticatorClient.login("admin", "admin", "localhost");
```

Comment:
Why do we need to make this `localhost`? This means we no longer take the property values coming from Deployment Properties.

Reply:
AuthenticatorClient is already initialized with the correct keyManagerURL as you see two line above.

`new AuthenticatorClient(keyManagerURL);`

The third parameter in #login is not really used as the endpoint. It was used for some unnecessary validations for this case.

Reply:
AWS deployment tests are passing - https://testgrid-live-dev.private.wso2.com/kasung-wso2am-2.6.0-aws-new-tests

K8S tests pass as well (this has test failures with mysql db due to other reasons) - https://testgrid-live-dev.private.wso2.com/wum-sce-test-helm-wso2am-2.6.0-full